main/main_views/order_detail.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import generics
+from main.models import Order
+from main.serializers import OrderDetailSerializer
+
+
+# TODO: Change to Retrieve API View
+class OrderDetail(generics.RetrieveUpdateDestroyAPIView):
```

Comment:
Please make it a RetreiveAPIView and change the serializer to OrderSerializer. We don't need an order detail serializer.